buildx-bake.hcl

```diff
@@ -0,0 +1,84 @@
+variable "ENABLE_GIT_COMMAND" {
+    default= "true"
+}
+variable "VERSION" {
+    default= "workdir-dev"
+}
+
+variable "OUTPUT" {
+    default= "image"
+}
+
+variable "IMAGE_REGISTRY" {
+    default= "local"
+}
+
+variable "IMAGE_TAG" {
+    default= "latest"
+}
+
+group "default" {
+    targets = ["ccm-amd64","ccm-arm64","ccm-arm","cnm-amd64","cnm-arm64","cnm-arm"]
```

Comment:
Could this also support the Windows targets?

Reply:
sure